src/features/ShowMovie/ui/DatesNav.tsx

```diff
@@ -60,7 +63,7 @@ export const DatesNav: FC<DatesNavProps> = ({ dates, setActiveDate, activeDate }
                             isActive={isActive}
                             onClick={onDateClick}
                             wholeDate={date}
-                            key={convertedDate}
+                            key={convertedDate + day + month}
```

Comment:
is it unique value? can we have 2 items with the same combination of convertedDate + day + month?